marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-vis/graph-vis.tsx

```diff
@@ -571,25 +571,16 @@ const GraphVis: React.FC<Props> = (props) => {
           setCoords(newCoords);
           props.saveEntityCoords(nodes[0], positions.x, positions.y);
         }
+      } else {
+        let ids = Object.keys(coords);
+        // Take current positions and update with drag deltas
+        ids.forEach(id => {
+          let newX = coords[id].graphX + event.event.deltaX;
+          let newY = coords[id].graphY + event.event.deltaY;
+          props.saveEntityCoords(id, newX, newY); // Save to db
```

Comment:
@AkshayBajajML, right now we're doing an endpoint call for every node that is panned. @xnikhil08 added the ability to update the coordinates of all the entities with a single endpoint call, see here: https://github.com/marklogic/marklogic-data-hub/pull/6105/files#diff-b974bd3271e6146b0ccee23fc50c94a1586ac7f747951f06f25d8462e1c53a4dR129

Can we do the same when reacting to a pan? Build and updatedEntityDefinitions array and then make a updateSavedEntity() call?

Reply:
Thanks! for review and comment.
Yeah that is a good idea. I have implemented the mentioned changes and PR is ready for review again.